### ISSUE TITLE
Make the mapping of problem index to problem files less obscure

### ISSUE THREAD

**Issue by gehring** [tomsilver/pddlgym#18]
The mapping of problem index to problem files is not visible without diving into the code. Either we establish an explicit mapping of indices to pddl problem files, e.g., a yaml with a list of pddl file names or a dict of index and file names, or we keep the problem index private and remove it from the public API, e.g., replace `PDDLEnv.fix_problem_index(index)` with something like `PDDLEnv.fix_problem(problem_filepath)`.

**Comment by tomsilver** [comment]
Yup agreed. I like `fix_problem(problem_filepath)` as an option.
